nullstone.tf

```diff
@@ -3,6 +3,9 @@ terraform {
     ns = {
       source = "nullstone-io/ns"
     }
+    restapi = {
```

Comment:
Does it make sense to call it "restapi"? Seems like we would call it something like "db_admin". If we decided to follow this pattern for something else, I would imagine we would create a new provider with a different name.

Reply:
That's the name of the Terraform provider.

This line creates an alias named `restapi` to the provider and while it's possible to name it anything, it feels a bit wrong to do that in Terraform -- TF contributors use the resource prefix `restapi_` to search for docs on registry.terraform.io.